packages/next/src/server/config-shared.ts

```diff
@@ -188,6 +188,7 @@ export interface ExperimentalConfig {
   caseSensitiveRoutes?: boolean
   appDocumentPreloading?: boolean
   preloadEntriesOnStart?: boolean
+  /** @default true */
   strictNextHead?: boolean
```

Comment:
Since this was experimental I think we can remove it

Reply:
@timneutkens mentioned that some people may complain for SEO reasons even though extra attributes do not affect it. At least that's what happened in the past he said.

 If this comes up, and people don't provide evidence that `strictNextHead: true` is bad, we can offer `strictNextHead: false` as an escape hatch. Client-side updates to `next/head` children would be broken but at least it would fix their imagined SEO issues.

But practically, I want to remove this soon after Next.js 15 is out stable. vercel/front is running with `strictNextHead` for a while by that time so it should've had sufficient soak-time.